app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    vaccine_flag = False
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except VaccineError:
+            vaccine_flag = True
+    if vaccine_flag:
+        return "All friends should be vaccinated"
```

Comment:
You can return string "All friends should be vaccinated" inside except VaccineError block.

Reply:
But overall GJ! I also liked that you grabbed name of visitor in Cafe's visit method.Error's descriptions are also very good!

Reply:
@ZabFTFT is right, move return on 19 line below the Vaccine Error, no need to create usless variable